packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -127,9 +128,36 @@ const TradersHub = observer(() => {
         return <OrderedPlatformSections is_cfd_visible={false} is_options_and_multipliers_visible={true} />;
     };
 
+    const mobileTabletContent = (
+        <React.Fragment>
+            {is_mt5_allowed &&
+                (is_landing_company_loaded ? (
+                    <ButtonToggle
+                        buttons_arr={is_eu_user ? platform_toggle_options_eu : platform_toggle_options}
+                        className='traders-hub__button-toggle'
+                        has_rounded_button
+                        is_traders_hub={window.location.pathname === routes.traders_hub}
+                        name='platforn_type'
+                        onChange={platformTypeChange}
+                        value={selected_platform_type}
+                    />
+                ) : (
+                    <ButtonToggleLoader />
+                ))}
+            {!is_mt5_allowed && (
```

Comment:
You are checking twice `is_mt5_allowed` (here and in line 133), it can be done once in ternary. If you don't want to use ternary twice here, then `<ButtonToggle/>` and `<ButtonToggleLoader />` could be extracted into a separate constant. Please, ignore if irrelevant